internal/backend/post.go

```diff
@@ -123,3 +208,11 @@ func (d *delphisBackend) iterToPosts(ctx context.Context, iter datastore.PostIte
 
 	return posts, nil
 }
+
+func validateMentionedEntities(ctx context.Context, inputText string, entities []string) error {
+	tokens := regexp.MustCompile(`\<(.*?)\>`).FindAllStringSubmatch(inputText, -1)
```

Comment:
This should be (\d+) right? Like we wouldn't want to match anything other than a number, I think?

Reply:
yes